src/Illuminate/Http/Middleware/Cache.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Illuminate\Http\Middleware;
+
+use Closure;
+
+class Cache
+{
+    /**
+     * Add cache related HTTP headers.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @param  int|null  $maxAge
+     * @param  int|null  $sharedMaxAge
+     * @param  bool|null $public
+     * @param  bool|null $etag
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function handle($request, Closure $next, int $maxAge = null, int $sharedMaxAge = null, bool $public = null, bool $etag = false)
+    {
+        /**
+         * @var   \Symfony\Component\HttpFoundation\Response
```

Comment:
missing $response

Reply:
It has been removed by StyleCI actually (https://github.com/laravel/framework/pull/22389/commits/29a3e96ea9e3a7481a84c87212bbe64932d2cd58#diff-b3ff80d2f50088f0bfe3b4e15d654af9L23).